kfs-core/src/main/java/org/kuali/kfs/gl/businessobject/OriginEntryFull.java

```diff
@@ -214,12 +215,15 @@ protected String getValue(String line, int s, int e) {
         setChartOfAccountsCode(getValue(line, pMap.get(KFSPropertyConstants.CHART_OF_ACCOUNTS_CODE), pMap.get(KFSPropertyConstants.ACCOUNT_NUMBER)));
         setAccountNumber(getValue(line, pMap.get(KFSPropertyConstants.ACCOUNT_NUMBER), pMap.get(KFSPropertyConstants.SUB_ACCOUNT_NUMBER)));
 
-        // if chart code is empty while accounts cannot cross charts, then derive chart code from account number
-        AccountService acctserv = SpringContext.getBean(AccountService.class);
-        if (StringUtils.isEmpty(getChartOfAccountsCode()) && StringUtils.isNotEmpty(getAccountNumber()) && !acctserv.accountsCanCrossCharts()) {
+        if (StringUtils.isNotEmpty(getAccountNumber())) {
+            AccountService acctserv = SpringContext.getBean(AccountService.class);

```

Comment:
I like how Scott moved the call to get the service inside the if statement, avoiding an additional call if the account number is empty.